docs/execution-analysis/simulation.md

```diff
@@ -34,3 +34,24 @@ After the simulation runs, it prints the following:
 {% hint style="warning" %}
 There are some operations which are not supported in simulation yet. They will result in compilation failures. You can revert to simulation using graph execution using `circuit.graph(...)` instead of `circuit.simulate(...)`, which won't simulate FHE, but it will evaluate the computation graph, which is like simulating the operations without any errors due to FHE.
```

Comment:
```suggestion
Some operations are not yet supported in simulation and will result in compilation failures. In such cases, you can use graph execution with `circuit.graph(...)` instead of `circuit.simulate(...)`. This will evaluate the computation graph without simulating FHE errors.
```